tests/datavolume_test.go

```diff
@@ -119,10 +134,28 @@ var _ = Describe("DataVolume Integration", func() {
 				ExpectWithOffset(1, virtClient.CdiClient().CdiV1alpha1().DataVolumes(dv.Namespace).Delete(dv.Name, &metav1.DeleteOptions{})).To(Succeed())
 			}
 
+			handleWaitForFirstConsumerDv := func(dv *cdiv1.DataVolume, dvConsumerPod *k8sv1.Pod) *k8sv1.Pod {
+				dataVolumePhase := dv.Status.Phase
+				if dataVolumePhase == cdiv1.WaitForFirstConsumer && dvConsumerPod == nil {
+					// createConsumerPod, this will force
+					dvConsumerPod, err = virtClient.CoreV1().Pods(tests.NamespaceTestDefault).Create(renderConsumerPod(dv.Name))
```

Comment:
we don't need this anymore, right? This is what the vmi.go logic is doing with the temp pod?

Reply:
this is used in test in places where test creates Datavolume and waits for it to start import, but... maybe that logic is not needed -- checking

Reply:
I thought that in this test it might be needed, because it needs to have a DV with bad url and then it tests the eventual consistency of DV. But I see that the test does not need to wait for dv to be ready.